integration-test/src/test/java/org/apache/iotdb/db/it/IoTDBConfigNodeIT.java

```diff
@@ -0,0 +1,160 @@
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+@RunWith(IoTDBTestRunner.class)
+@Category({ClusterIT.class})
+public class IoTDBConfigNodeIT {
+
+  protected static String enableConfigNodeConsensusProtocolClass;
+  protected static String enableSchemaRegionConsensusProtocolClass;
+  protected static String enableDataRegionConsensusProtocolClass;
+
+  private final int retryNum = 30;
+
+  @Before
+  public void setUp() throws Exception {
+    enableConfigNodeConsensusProtocolClass =
+        ConfigFactory.getConfig().getConfigNodeConsesusProtocolClass();
+    enableSchemaRegionConsensusProtocolClass =
+        ConfigFactory.getConfig().getSchemaRegionConsensusProtocolClass();
+    enableDataRegionConsensusProtocolClass =
+        ConfigFactory.getConfig().getDataRegionConsensusProtocolClass();
+
+    ConfigFactory.getConfig()
+        .setConfigNodeConsesusProtocolClass("org.apache.iotdb.consensus.ratis.RatisConsensus");
+    ConfigFactory.getConfig()
+        .setSchemaRegionConsensusProtocolClass("org.apache.iotdb.consensus.ratis.RatisConsensus");
+    ConfigFactory.getConfig()
+        .setDataRegionConsensusProtocolClass("org.apache.iotdb.consensus.ratis.RatisConsensus");
+
+    EnvFactory.getEnv().initBeforeClass();
+  }
+
+  @After
+  public void tearDown() {
+    EnvFactory.getEnv().cleanAfterClass();
+    ConfigFactory.getConfig()
+        .setConfigNodeConsesusProtocolClass(enableConfigNodeConsensusProtocolClass);
+    ConfigFactory.getConfig()
+        .setSchemaRegionConsensusProtocolClass(enableSchemaRegionConsensusProtocolClass);
+    ConfigFactory.getConfig()
+        .setDataRegionConsensusProtocolClass(enableDataRegionConsensusProtocolClass);
+  }
+
+  private TShowClusterResp getClusterNodeInfos(
+      IConfigNodeRPCService.Iface client, int configNodeNum, int dataNodeNum)
+      throws TException, InterruptedException {
+    TShowClusterResp clusterNodes = null;
+    for (int i = 0; i < retryNum; i++) {
+      clusterNodes = client.showCluster();
+      Thread.sleep(1000);
+
+      if (clusterNodes.getConfigNodeListSize() == configNodeNum
+          && clusterNodes.getDataNodeListSize() == dataNodeNum) break;
+    }
+
+    if (clusterNodes.getConfigNodeListSize() != configNodeNum
+        || clusterNodes.getDataNodeListSize() != dataNodeNum) {
+      fail("getClusterNodeInfos failed");
+    }
+
+    return clusterNodes;
+  }
+
+  @Test
+  public void removeAndStopConfigNodeTest() {
+    TShowClusterResp clusterNodes;
+    TSStatus status;
+
+    List<ConfigNodeWrapper> configNodeWrappers = EnvFactory.getEnv().getConfigNodeWrapperList();
+
+    try (SyncConfigNodeIServiceClient client =
+        (SyncConfigNodeIServiceClient) EnvFactory.getEnv().getConfigNodeConnection()) {
+      // add ConfigNode
+      for (int i = 0; i < 2; i++) {
+        ConfigNodeWrapper configNodeWrapper =
+            new ConfigNodeWrapper(
+                false,
+                configNodeWrappers.get(0).getIpAndPortString(),
```

Comment:
I run this test meets npe.
![image](https://user-images.githubusercontent.com/6756545/181218411-96105c6b-03da-45e5-9379-e70b73dbc290.png)
`EnvFactory.getEnv().getConfigNodeWrapperList();` returns null;

Reply:
the test should be run in Cluster Mode
![16915028372f52396662a2f7363ffd0](https://user-images.githubusercontent.com/42286868/181227075-33920937-3656-41e8-b6a7-29ea7fdfbaf6.png)